src/Symfony/Component/Messenger/Bridge/GooglePubSub/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "symfony/google-pubsub-messenger",
+    "type": "symfony-bridge",
+    "description": "Symfony Google Pub/Sub extension Messenger Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "ext-json": "*",
+        "google/cloud-pubsub": "^1.0",
+        "symfony/messenger": "^4.3|^5.0",
+        "symfony/service-contracts": "^1.1|^2"
```

Comment:
You're not using the service contracts anywhere, are you? Please only add dependencies you actually use (same for the `require-dev` part).